src/modules/dashboard/dashboard.scss

```diff
@@ -66,4 +66,8 @@
     padding: 0.125rem 1rem;
     transition: all 0.2s ease;
   }
-}
\ No newline at end of file
+}
+
+.container {
```

Comment:
I think the name container would collide with multiple classnames. 
Make class selection more specific.